### ISSUE TITLE
can't import yaml

### ISSUE THREAD

**Issue by white-hat** [SublimeLinter/SublimeLinter-pyyaml#3]
Can you please explain

> Before using this plugin, you must ensure that pyyaml is installed for Python 3 on your system, and available for direct import in Sublime Text 3. 

AFAIK subl runs in it's own python 3.3 environment, so how can Import yaml that was installed to system python3 environment in subl?

**Comment by NotSqrt** [comment]
SublimeText/Linter magically adds some system python folders to the path.

It works identically with pep257, pep8 plugins, etc..

In SublimeText console, if you do
```python
>>> import sys
>>> sys.path
```
you should see system folders.

**Comment by white-hat** [comment]
>>> import sys
>>> sys.path
['/Applications/Sublime Text.app/Contents/MacOS', '/Applications/Sublime Text.app/Contents/MacOS/python3.3.zip', '/Users/itrushchenko/Library/Application Support/Sublime Text 3/Packages', '/Users/itrushchenko/Library/Application Support/Sublime Text 3/Packages/bz2/st3_osx_x64']

**Comment by NotSqrt** [comment]
You do have a python3 installed on your system, right ?

**Comment by white-hat** [comment]
```
itrushchenko@AF6202  ~  python3
Python 3.5.0a2 (default, Mar 18 2015, 17:41:14)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

**Comment by smsrkr** [comment]
Don't know about Mac, but on Windows, this problem goes away if Python3 paths come before python2 in %PATH% env.var.
`%PATH% = ...;c:\python2;c:\python2\scripts;c:\python3;c:\python3\scripts;...`
```python
`SublimeLinter: import of yaml module in pyyaml failed, linter will not run using built in python `
`SublimeLinter: pyyaml activated: <builtin> `
>>> import sys; sys.path
['P:\\ath\\to\\many st packages', '...', 'P:\\ath\\to\\SublimeText', 'P:\\ath\\to\\SublimeText/python3.3.zip', 'P:\\ath\\to\\more st packages']
```
`%PATH% = ...;c:\python3;c:\python3\scripts;c:\python2;c:\python2\scripts;...`
```python
`SublimeLinter: pyyaml activated: <builtin> `
>>> import sys; sys.path
['P:\\ath\\to\\many st packages', 'P:\\ath\\to\\SublimeText', 'P:\\ath\\to\\SublimeText/python3.3.zip', '...', 'C:\\python3\\python35.zip', 'C:\\python3\\DLLs', 'C:\\python3\\lib', 'C:\\python3', 'C:\\python3\\lib\\site-packages', 'P:\\ath\\to\\more st packages']
```

**Comment by NotSqrt** [comment]
Nice catch !

**Comment by mew1033** [comment]
I actually don't have python 3 installed on my system. Don't use it for anything as all my work is done in python 2. I didn't want to install it just for pyyaml, so I cheated. ST3 uses python 3.3, so I downloaded the 64 bit windows binary from [here](http://pyyaml.org/wiki/PyYAML) (make sure to grab the python 3.3 version. Running it doesn't work because I don't have python 3 installed. I just opened the exe with 7zip, pulled out the `yaml` folder, and dumped it into the` python3.3.zip` file in `Program Files`.
Works great!
